pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -980,6 +1027,18 @@ func (p *MasqueradePodInterface) createNatRulesUsingNftables(proto iptables.Prot
 	return nil
 }
 
+func (p *MasqueradePodInterface) createVirtualNetworkingDevice(deviceName string, isMultiqueue bool, launcherPID int) error {
+	err := Handler.CreateTapDevice(deviceName, isMultiqueue, launcherPID)
+	if err == nil {
+		p.vif.TapDevice = deviceName
+	}
+	return err
+}
+
+func (m *MasqueradePodInterface) configureVirtualNetworkingDevice() error {
+	return Handler.BindTapDeviceToBridge(m.vif.TapDevice, m.bridgeInterfaceName)
+}
+
```

Comment:
Since this is the same as the `createVirtualNetworkingDevice` of the bridge binding, could we merge them and treat it as a function instead of a method?

Reply:
Good catch.

Done.